Source/Core/VideoBackends/OGL/TextureConverter.h

```diff
@@ -10,6 +10,8 @@
 #include "VideoCommon/TextureDecoder.h"
```

Comment:
This should be able to be moved into the cpp file considering the added forward declaration, no?